Language_Parser/actionVerbs.py

```diff
@@ -762,8 +784,9 @@ def goRoomHelper(roomInfo, currentRoom):
     if len(roomInfo) == 0 or len(roomInfo) > 2:
         return None
     if roomInfo[0] in ["stairs", "staircase", "upstairs", "downstairs",
-                       "up", "down"]:
-        return goStairsHelper(roomInfo[0], currentRoom.getName())
+                       "up", "down", "hatch", "ceiling"]:
+        i = goStairsHelper(roomInfo[0], currentRoom.getName())
+        return i
```

Comment:
Why does this need to get assigned to `i` if the next line is `return i`? Could return result directly as it was before

Reply:
No reason. Just had changed it when I was testing something and forgot to change it back. Fixed.